src/prefect/tasks/dbt/dbt_cloud_utils.py

```diff
@@ -26,7 +27,7 @@
     "https://{apiDomain}/api/v2/accounts/{accountId}/runs/{runId}/artifacts/{path}"
 )
 
-dbt_cloud_artifact_paths = ("manifest.json", "run_results.json", "catalog.json")
+USER_AGENT_HEADER = {"user-agent": "prefect-{v}".format(v=prefect.__version__)}
```

Comment:
Nit; not too familiar with user-agent, but I was wondering would it look better if it was:
`Prefect 1.0.0`
or
`prefect 1.0.0`
or
`prefect v1.0.0
vs
`prefect-1.0.0`

Reply:
Should we add platform too? 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
"User-Agent: Mozilla/5.0 (<system-information>) <platform> (<platform-details>) <extensions>"

Reply:
Should we also add flow name?

Reply:
The user-agent header is only used by dbt for tracking purposes so that they know the amount of traffic is coming from Prefect users. I don't think users will be able to see this so we have other plans to expose useful info to users in the dbt Cloud console such as which flow run triggered a given job.

Reply:
On a separate note, any reason we're not just doing a f-string?
```suggestion
USER_AGENT_HEADER = {"user-agent": f"prefect-{prefect.__version__}"}
```